site/content/en/docs/manual/advanced/automatic-annotation.md

```diff
@@ -2,54 +2,115 @@
 title: 'Automatic annotation'
 linkTitle: 'Automatic annotation'
 weight: 16
-description: 'Guide to using the automatic annotation of tasks.'
+description: 'Automatic annotation of tasks'
 ---
 
-Automatic Annotation is used for creating preliminary annotations.
-To use Automatic Annotation you need a DL model that can be deployed by a CVAT administrator.
-You can find the list of available models in the `Models` section.
+Automatic annotation in CVAT is a tool that you can use
+to automatically pre-annotate your data with pre-trained
+and pre-installed [Models](#models).
```

Comment:
This "pre-trained and pre-installed" wording suggests that "pre-trained" and "pre-installed" are alternatives, which they aren't. All models that are used for annotations are necessarily pre-trained. I would suggest something like this:

> Automatic annotation in CVAT is a tool that you can use
to automatically pre-annotate your data with pre-trained models.
>
> CVAT can use models from the following sources:
>
> * Pre-installed models
> * Models published in Hugging Face and Roboflow.
> * Self-hosted models deployed with Nuclio
>
> [link to the appropriate section for each of those]
>
> [table comparing self-hosted CVAT with CVAT Cloud]